docs/user/logging.md

```diff
@@ -0,0 +1,104 @@
+# Logging
+
+All log messages from `pypdf` go through Python’s standard `logging` library under the logger name `pypdf`. This gives you full control over verbosity, whether you want detailed debug information or only critical errors.
+
+## Filtering logs
+
+You can adjust the minimum log level of pypdf as follow:
+
+```py
+import logging
+from pypdf import PdfReader
+
+
+logger = logging.getLogger("pypdf")
```

Comment:
Does this cover the submodules of *pypdf* as well, id est when issuing a `logger.warning` in `pypdf._reader` for example?